internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -42,12 +41,12 @@ func TestAccClusterAdvancedCluster_basicTenant(t *testing.T) {
 		CheckDestroy:             acc.CheckDestroyCluster,
 		Steps: []resource.TestStep{
 			{
-				Config: acc.ConvertAdvancedClusterToTPF(t, configTenant(projectID, clusterName)),
-				Check:  checkTenant(projectID, clusterName),
+				Config: acc.ConvertAdvancedClusterToSchemaV2(t, configTenant(projectID, clusterName)),
+				Check:  checkTenant(true, projectID, clusterName),
```

Comment:
@EspenAlbert  as we need to have info about schemav2 inside checks func, WDYT about also moving ConvertAdvancedClusterToSchemaV2 into config funcs?

Now that mocked tests will be the acc tests themselves, I don't think it's so useful to have ConvertAdvancedClusterToSchemaV2 calls here, and it's more consistent do it inside config funcs as in check funcs.

Reply:
Yeah, I like that idea. That will minimize the calls to `acc.ConvertAdvancedClusterToSchemaV2` and make the test steps nicer